### PR TITLE
Add hyper-bright-blue theme

### PR DESCRIPTION
**Pull request by vicnicius**
Adds https://github.com/vicnicius/hyper-bright-blue theme.

_Preview of the theme_

### CODE REVIEW

**Comment by vassbence** [comment]
Hey @vicnicius, the theme submission guidelines have been recently updated, you can [find them on the website](https://hyper-site-fi8ugu8o4.now.sh/store/submit). Your PR looks good, just move the preview image with the exact name of your theme to `/public/store` the repo as mentioned in the guide.

**Comment by vicnicius** [comment]
Hello, @vassbence. Sorry, I overlooked this bit of the docs. I amended the commit with the preview image added.

Thanks!

File: plugins.json [hunk 1]
```diff
@@ -620,5 +620,22 @@
     "type": "plugin",
     "preview": "https://github.com/jZhangTk/hyper-tabs-autohide/raw/master/hyper-tabs-autohide.gif",
     "dateAdded": "1583633829078"
+  },
+  {
+    "name": "hyper-tomorrow-night-blue",
```
Comment (vassbence) [inline plugins.json:625]:
```suggestion
    "name": "hyper-bright-blue",
```

File: plugins.json [hunk 2]
```diff
@@ -620,5 +620,22 @@
...
+    "type": "theme",
+    "description": "An interpretation of the Night Blue version of the classic Night Theme.",
+    "preview": "https://github.com/vicnicius/hyper-bright-blue/raw/master/assets/preview.png",
```
Comment (vassbence) [inline plugins.json:628]:
```suggestion
```

**Comment by vassbence** [comment]
LGTM thanks @vicnicius. CC @timothyis  @pacocoursey, deployment failed probably due to the 3 rebuils quickly after each other, we could discuss disabling ssg in previews, but the theme is tested and works, ready for merge 👍

**Comment by vicnicius** [comment]
Thanks, @vassbence. Sorry for spamming CI.

**Pull request closed without merging**